utils/docker/Dockerfile-mockbuild.centos.7

```diff
@@ -27,32 +28,33 @@ RUN echo "$USER:$PASSWD" | chpasswd
 RUN usermod -a -G mock $USER
 
 # mock in Docker needs to use the old-chroot option
-RUN echo "config_opts['use_nspawn'] = False" >> /etc/mock/site-defaults.cfg
+RUN grep use_nspawn || \
+    echo "config_opts['use_nspawn'] = False" >> /etc/mock/site-defaults.cfg
 
 RUN echo -e "config_opts['yum.conf'] += \"\"\"\n\
 [openpa]\n\
 name=openpa\n\
-baseurl=https://build.hpdd.intel.com/job/daos-stack/job/openpa/job/master/lastSuccessfulBuild/artifact/artifacts/\n\
+baseurl=https://build.hpdd.intel.com/job/daos-stack/job/openpa/job/PR-4/lastSuccessfulBuild/artifact/artifacts/centos7/\n\
```

Comment:
This isn't ready for review yet if you're using a PR url?

Reply:
Yeah, not ready for review yet.  Still building the stack.